src/assets/scss/material.orcid.overwrites.scss/button.scss

```diff
@@ -16,6 +16,20 @@ button.mat-button-wrap-text {
   }
 }
 
+button.mat-button-wrap-long-texts {
```

Comment:
What was the issue with the existing `mat-button-wrap-text`? 

It seems to me like it handles the long text properly 
![image](https://user-images.githubusercontent.com/2119626/83896542-938d1200-a711-11ea-975a-475b2b5eeb48.png)

Reply:
The problem is that doesn't work for all scenarios.

![image](https://user-images.githubusercontent.com/8917901/83897634-72312380-a71b-11ea-87c1-d1430a07bb9e.png)

But I will then refactor `mat-button-wrap-text` instead of creating a new one.